test/unit/component/footer.nuxt.spec.ts

```diff
@@ -0,0 +1,9 @@
+import {it, expect} from "vitest"
+import {mountSuspended} from "vitest-environment-nuxt/utils"
+import {Footer} from "#components"
+
+it("Check footer display", async () =>{
+  const component = await mountSuspended(Footer)
+  expect(component.findComponent("GithubIcon")).toBeTruthy()
+  expect(component.findComponent("HeartIcon")).toBeTruthy()
+})
```

Comment:
Check for the license text 
You should also wrap the tests in a `describe()`
On develop look into `test/unit/component/calendarInfoButton.nuxt.spec.ts` for more info